xsd/netex_part_2/part2_occupancy/netex_oc_occupancy_version.xsd

```diff
@@ -0,0 +1,242 @@
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="OnboardCount" type="NumberOfPassengers" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Total number of passengers on-board after departing the STOP POINT.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="SpecialPlacesOccupied" type="NumberOfPassengers" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Total number of special places, e.g. seats for the disabled or lounge seats, that are occupied after departing the STOP POINT.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="PushchairsOnboardCount" type="NumberOfPassengers" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Total number of pushchairs on-board after departing the STOP POINT.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="WheelchairsOnboardCount" type="NumberOfPassengers" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Total number of wheelchairs on-board after departing the STOP POINT.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="PramsOnboardCount" type="xsd:nonNegativeInteger" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Total number of prams on-board after departing the STOP POINT.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="BicycleOnboardCount" type="xsd:nonNegativeInteger" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Total number of bicycles on-board, i.e., number of bicycle racks that are occupied after departing the STOP POINT.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+		</xsd:sequence>
+	</xsd:group>
+	<xsd:complexType name="GroupReservationStructure">
+		<xsd:annotation>
+			<xsd:documentation>Used to specify that a travel group has booked a section of the vehicle for a part of the journey, and if so under what name. (since SIRI 2.1)</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="NameOfGroup" type="siri:NaturalLanguageStringStructure">
+				<xsd:annotation>
+					<xsd:documentation>Name for which the travel group has made the reservation.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="NumberOfReservedSeats" type="NumberOfPassengers">
+				<xsd:annotation>
+					<xsd:documentation>Number of seats that the group has booked.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+		</xsd:sequence>
+	</xsd:complexType>
+	<xsd:element name="Occupancy_" type="Occupancy_VersionStructure" abstract="true" substitutionGroup="DataManagedObject">
+		<xsd:annotation>
+			<xsd:documentation>Dummy supertype for Occupancy.</xsd:documentation>
+		</xsd:annotation>
+	</xsd:element>
+	<xsd:element name="Occupancy" abstract="true" substitutionGroup="Occupancy_">
```

Comment:
Why _abstract="true"_ ... Don't we need a concrete one to put it as first class element in Frame in order to Ref it ?
If we change it to _abstract="false"_ then the different (below)
           _<xsd:element name="Occupancy" type="Occupancy_VersionStructure"/>_
can be replaced by
           _<xsd:element ref="Occupancy"/>_